src/write_eval_config.R

```diff
@@ -128,7 +128,7 @@ write_eval_config <- function(locations, forecast_dates,
     baseline_score_table_dir = baseline_score_table_dir,
     output_dir = output_dir,
     hub_subdir = hub_subdir,
-    retro_rt_dir = retro_rt_dir,
+    retro_rt_path = retro_rt_path,
```

Comment:
Can you explain this change? The code has many references to `retro_rt_path`, but none to `retro_rt_dir`. https://github.com/search?q=repo%3ACDCgov%2Fwastewater-informed-covid-forecasting%20retro_rt&type=code

Reply:
Right, that's why I am changing it to `retro_rt_path`. This is pointing to a full path where the Rt draws (including the file and its extension) will be, so it should be called a path. We tried to call everything a `dir` if it points to a folder.

Reply:
Not sure why I didn't understand that yesterday 🙃

Thanks!